pkg/cmd/config.go

```diff
@@ -812,13 +812,15 @@ addons:
 #     bastion: '4.3.2.1'
 #     bastionPort: 22  # can be left out if using the default (22)
 #     bastionUser: 'root'  # can be left out if using the default ('root')
+#     bastionHostPublicKey: "AAAAC3NzaC1lZDI1NTE5AAAAIGpmWkI5dl7GB3E1hB9LDuju87x9hX5Umw9fih+xXNU+"
 #     sshPort: 22 # can be left out if using the default (22)
 #     sshUsername: root
 #     # You usually want to configure either a private key OR an
 #     # agent socket, but never both. The socket value can be
 #     # prefixed with "env:" to refer to an environment variable.
 #     sshPrivateKeyFile: '/home/me/.ssh/id_rsa'
 #     sshAgentSocket: 'env:SSH_AUTH_SOCK'
+#     sshHostPublicKey: "AAAAC3NzaC1lZDI1NTE5AAAAIPwEDvXiKfvXrysf86VW5dJTKDlQ09e2tV0+T3KeFKmI"
```

Comment:
Can you add a comment describing this field similar to what we have for other fields?